zachG add UDS setup to Kubernetes trace collection docs (#12258)
DataDog/documentation: content/en/agent/kubernetes/apm.md

## Setup

> To enable trace collection with your Agent, follow the instructions below. You can configure the agent to intake traces via UDP or Unix Domain Socket:

Reviewer: ```suggestion
To enable trace collection with your Agent, follow the instructions below. You can configure the Agent to intake traces via UDP or Unix Domain Socket:
```

                        fieldPath: status.hostIP
    ```

    ```yaml
        apiVersion: apps/v1
        kind: Deployment
         # ...
            spec:
              containers:
              - name: "<CONTAINER_NAME>"
                image: "<CONTAINER_IMAGE>"/"<TAG>"
                env:
                  - name: DD_AGENT_HOST
                    valueFrom:
                      fieldRef:
                        fieldPath: status.hostIP
    ```

3. **Configure your application tracers to emit traces**: Point your application-level tracers to where the Datadog Agent host is using the environment variable `DD_AGENT_HOST`. Refer to the [language-specific APM instrumentation docs][3] for more examples.
{{% /tab %}}
{{% tab "Unix Domain Socket" %}}

> If sending traces to the agent via Unix Domain Socket you need to mount the directory the socket is in to the application container and specify the path with `DD_TRACE_AGENT_URL`.

Reviewer: ```suggestion
If you are sending traces to the Agent by using Unix Domain Socket, mount the directory the socket is in to the application container and specify the path with `DD_TRACE_AGENT_URL`.
```

    ```yaml
        apiVersion: apps/v1
        kind: Deployment
        # ...
            spec:
              containers:
              - name: "<CONTAINER_NAME>"
                image: "<CONTAINER_IMAGE>"/"<TAG>"
                imagePullPolicy: Never
                volumeMounts:
                - name: apmsocketpath
                  mountPath: /var/run/datadog
                ports:
                - containerPort: 9390
                env:
                - name: DD_TRACE_AGENT_URL
                  value: 'unix:///var/run/datadog/apm.socket'
                # ...
              volumes:
                - hostPath:
                    path: /var/run/datadog/
                  name: apmsocketpath
    ```

> 3. **Configure your application tracers to emit traces**: Point your application-level tracers to where the Datadog Agent host is using the environment variable `DD_APM_RECEIVER_SOCKET` or for Helm `apm.socketPath`. For Helm and Operator the default for this value is `"/var/run/datadog/apm.socket"`. Some tracers do not yet support sending via UDS, refer to the [language-specific APM instrumentation docs][3] to see if this is supported for the tracr you're using.

Reviewer: ```suggestion
3. **Configure your application tracers to emit traces**: Point your application-level tracers to where the Datadog Agent host is using the environment variable `DD_APM_RECEIVER_SOCKET` or for Helm `apm.socketPath`. For Helm and Operator the default for this value is `"/var/run/datadog/apm.socket"`. Some tracers do not support Unix Domain Socket. Refer to the [language-specific APM instrumentation docs][3] to see if this is supported for your tracer.
```
